randovania/games/samus_returns/json_data/header.txt

```diff
@@ -35,34 +35,44 @@ Templates
       Morph Ball and Spider Ball
 
 * Defeat Alpha Metroid:
-      Ice Beam or Missile or Super Missile ≥ 5 or Shoot Beam Burst
+      Any of the following:
+          Ice Beam or Missile or Shoot Beam Burst
+          Super Missile ≥ 5 and Shoot Super Missile
```

Comment:
why is the super requirement higher than the missile requirement?

Reply:
I wasn't sure if they gave Super drops mid fight. But this is only relevant when missiles are shuffleable. I'll just make it 1.